README.md

```diff
@@ -654,7 +657,7 @@ the `frontend` is exposed to the host and beyond.
 
 To do this, we're going to remove host network access from the `frontend`
 service to allow external incoming network traffic into it, as well as ensure
-only incoming requests from the same bridge (ie. the `frontend` service) are
+only incoming requests from the same bridge (eg. the `frontend` service) are
```

Comment:
```suggestion
only incoming requests from the same bridge (i.e. the `frontend` service) are
```
nit but I'd say i.e. is the right use there as we are referring to the actual `frontend` service here.

Reply:
Yeah, fair enough. I kind of assumed they were interchangeable, but after closer examination `i.e.` is "in other words" and `e.g.` is "for example" :)